package.json

```diff
@@ -10,7 +10,7 @@
     "lint": "eslint index.js ./**/*.{js,ts}",
     "lint:fix": "eslint index.js ./**/*.{js,ts} --fix",
     "format": "prettier --config .prettierrc --write index.js ./**/*.{js,json,ts}",
-    "gentoc": "markdown-toc -i README.md",
+    "gentoc": "markdown-toc -i --bullets=- README.md",
```

Comment:
What's this --bullets for?

Reply:
Oh it determines the bullet format for README table of contents, currently there's "+/*/-" this standardizes all to "-"